scripts/create_partitions/table_names.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# TABLE_NAMES=( \
+#     schema1.table1
+#     schema1.table2
+#     schema2.table1
+#     schema2.table2
+# )
+
+# It should include the schema 
+
```

Comment:
I think this would be simpler as a `.txt` file. Like this:
```suggestion
schema1.table1
schema1.table2
schema2.table1
schema2.table2
```

[Here](https://github.com/jampp/data-docker-images/blob/e08d4e328e94c1c93a3d63707210c83b259a598e/minio/create_buckets.sh#L8) you have an example of reading each line in a .txt file and executing a command.